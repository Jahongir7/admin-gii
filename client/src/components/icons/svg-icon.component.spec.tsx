import { customRender } from '@test-utils';
import { screen } from '@testing-library/react';
import { SvgIcon } from './svg-icon.component';

describe('Atoms Context', () => {
  test('Should render Icon component', async () => {
    const { asFragment } = customRender(
      <SvgIcon data-testid='svg-icon-component' iconName={'badge-icon'} />
    );
    expect(asFragment()).toMatchSnapshot();
    expect(screen.getByTestId('svg-icon-component')).toBeInTheDocument();
  });
});