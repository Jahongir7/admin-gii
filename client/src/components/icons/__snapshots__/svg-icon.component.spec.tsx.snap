// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Atoms Context Should render Icon component 1`] = `
<DocumentFragment>
  .c0 {
  width: 2rem;
  height: 2rem;
  fill: #000000;
  stroke: none;
  -webkit-transition-duration: 200ms;
  transition-duration: 200ms;
}

.c0[disabled] {
  opacity: 0.2;
}

<svg
    class="c0"
    data-testid="svg-icon-component"
  >
    <use
      xlink:href="sprite.svg#badge-icon"
    />
  </svg>
</DocumentFragment>
`;
